palette.cpp

```diff
@@ -5,26 +5,34 @@
 //
 // Contains the implementation of the Palette class defined in palette.h.
 
-const std::vector<CRGB> Palette::Rainbow = 
-{
-    CRGB(255, 0, 0),   // Red
-    CRGB(255, 165, 0), // Orange
-    CRGB(0, 255, 0),   // Green
-    CRGB(0, 255, 255), // Cyan
-    CRGB(0, 0, 255),   // Blue
-    CRGB(128, 0, 128)  // Purple
-};
+// Rainbow - 8 colors for smoother transitions in a performant power of 2 size
 
-const std::vector<CRGB> Palette::ChristmasLights = 
-{
+template<>
+const Palette<8> Palette<8>::Rainbow({
```

Comment:
These static `Palette<N>` class member variable palettes are not used anywhere; the `PaletteEffect<N>` instances created in main.cpp use the palettes in the `StandardPalettes` namespace as declared in pixeltypes.h. I think one set of standard palettes should stay, and the other should go.